extension/WebJobs.Extensions.RabbitMQ.Tests/RabbitMQAsyncCollectorTests.cs

```diff
@@ -16,7 +16,9 @@ public class RabbitMQAsyncCollectorTests
         [Fact]
         public async Task AddAsync_AddsMessagesToQueue()
         {
-            var batchLock = new object();
+#pragma warning disable SA1000
+            object batchLock = new();
+#pragma warning restore SA1000
```

Comment:
Disabled warning as StyleCop was reporting error for valid C# 9 syntax. I can check afterwards if there is a beta/future version of StyleCop in making that will understand the new syntax (pun intended).